src/Comanda/Comanda.java

```diff
@@ -0,0 +1,135 @@
+package Comanda;
+
+import Persoane.Client;
+import Persoane.Curier.Curier;
+import Persoane.Curier.CurierBiciclist;
+import Persoane.Curier.CurierSofer;
+import Restaurant.Restaurant;
+
+import java.util.*;
+
+public class Comanda {
+
+    private int id;
+    private Client client;
+    private Curier curier;
+    private CurierSofer curierSofer;
+    private CurierBiciclist curierBiciclist;
+    private Restaurant restaurant;
+    private Date plasareComanda;
+    private double totalPlata;
+    ArrayList<ComandaFelMancare> continutComanda;
```

Comment:
use abstract reference, List instead of ArrayList, just when instantiating use ArrayList object